llama-index-integrations/node_parser/llama-index-node-parser-relational-dashscope/llama_index/node_parser/dashscope/base.py

```diff
@@ -36,6 +36,9 @@ class DashScopeJsonNodeParser(BaseElementNodeParser):
         description="language of tokenizor, accept cn, en, any. Notice that <any> mode will be slow.",
     )
 
+    def __init__(self, dashscope_api_key: Optional[str] = None):
+        self.dashscope_api_key = dashscope_api_key or os.environ.get("DASHSCOPE_API_KEY")
```

Comment:
I think we should either set this as a `Field` like the others or set as a `PrivateAttr` and rename to `self._dashscope_api_key`.

Something like this: https://github.com/run-llama/llama_index/blob/f1965dfad3fbf9ff1e97e1948a4ed8604c9c266b/llama-index-integrations/llms/llama-index-llms-deepinfra/llama_index/llms/deepinfra/base.py#L100

Reply:
Modified the dashscope_api_key to PrivateAttr()